tests/algorithms/algorithm_settings.py

```diff
@@ -150,6 +150,7 @@
         }
     },
     WeightStandardization: simple_vision_settings,
+    GyroDropout: None, #Dropout settings needed
```

Comment:
This will put xfail around any tests that use GyroDropout form algorithm settings. 
```suggestion
    GyroDropout: None,  # Dropout settings needed
```

Reply:
Minor style nit to be consistent